mono_repo/lib/src/travis_command.dart

```diff
@@ -24,6 +24,7 @@ class TravisCommand extends Command {
 }
 
 Future generateTravisConfig({String rootDirectory}) async {
+  rootDirectory ??= p.current;
```

Comment:
I think I only pass the directory into getTravisConfig, right? Which handles null...

Not sure looking here...

Reply:
We also use this here, so the test was passing (we pass in a rootDirectory) but it actually was failing  when you went through the real command.

Reply:
Riiight...